ArgoProject/src/scenes/TutorialLevelScene.cpp

```diff
@@ -24,11 +40,19 @@ app::sce::TutorialLevelScene::TutorialLevelScene(SceneType & sceneManagerType)
 
 void app::sce::TutorialLevelScene::start()
 {
-	auto const & entities = fact::sce::TutorialLevelSceneFactory().create();
-	if constexpr (DEBUG_MODE)
+	m_registry.construction<comp::Layer>().disconnect<&app::sys::RenderSystem::onLayerConstruction>();
+	auto && entities = fact::sce::TutorialLevelSceneFactory().create();
```

Comment:
Go to `BaseScene` create a `protected` member function `std::vector<app::Entity> createEntities(fact::EntitiesFactory & entitiesFactory)`
*inside the function*
```cpp
	m_registry.construction<comp::Layer>().disconnect<&app::sys::RenderSystem::onLayerConstruction>();
	auto && entities = entitiesFactory.create();
	app::sys::RenderSystem::onLayerConstruction(m_registry, 0);
	m_registry.construction<comp::Layer>().connect<&app::sys::RenderSystem::onLayerConstruction>();
	return entities;
```

*in `void app::sce::TutorialLevelScene::start()`*
```suggestion
	auto entities = BaseScene::createEntities(fact::sce::TutorialLevelSceneFactory());
```